packages/plugins/Gfi/src/store/getters.ts

```diff
@@ -191,7 +182,7 @@ const getters: PolarGetterTree<GfiState, GfiGetters> = {
         )
           .filter(isVisible)
           .map((feature) => {
-            feature.set('_gfiLayerId', layer.get('id'))
+            feature.set('_gfiLayerId', layer.get('id'), true)
```

Comment:
Why should the update be silent? Or rather: Why wasn't it done before and only now became relevant?

Reply:
Since changing a feature's attributes may result in effects on the style and clustering, doing this loud results in a cluster recomputation and rerender. It became relevant on trying to use the productive service.

Reply:
This seems like information I would gladly stumble upon if we ever have to refactor this!

Reply:
https://github.com/Dataport/polar/pull/42/commits/dd558a4ef2a6219dc82e6ebb51b511f84547d843 De-arcaned.